examples/http-server-using-redis/main_test.go

```diff
@@ -108,7 +94,7 @@ func TestRedisPipelineHandler(t *testing.T) {
 	mock.ExpectSet("testKey1", "testValue1", time.Minute*5).SetErr(testutil.CustomError{ErrorMessage: "redis get error"})
 	mock.ClearExpect()
 
-	req, _ := http.NewRequestWithContext(context.Background(), http.MethodGet, fmt.Sprint("http://localhost:", httpPort, "/handle"), bytes.NewBuffer([]byte(`{"key":"value"}`)))
+	req, _ := http.NewRequestWithContext(context.Background(), http.MethodGet, fmt.Sprint("http://localhost:", configs.HTTPPort, "/handle"), bytes.NewBuffer([]byte(`{"key":"value"}`)))
```

Comment:
Isn't this one the configs.HTTPHost ?


```suggestion
	req, _ := http.NewRequestWithContext(context.Background(), http.MethodGet, configs.HTTPHost+ "/handle"), bytes.NewBuffer([]byte(`{"key":"value"}`)))
```

Reply:
yes, it was on oversight. fixing it.